README.md

```diff
@@ -18,11 +18,11 @@ Full [installation](https://cfpb.github.io/cfgov-refresh/installation/)
 and [usage](https://cfpb.github.io/cfgov-refresh/usage/) instructions
 are available in [our documentation](https://cfpb.github.io/cfgov-refresh).
 
-Ensure that Elasticsearch and MySQL are installed and that MySQL is
-either running or runnable by our
-[backend.sh script](https://github.com/cfpb/cfgov-refresh/blob/master/backend.sh#L41)
-and our
-[runserver.sh script](https://github.com/cfpb/cfgov-refresh/blob/master/runserver.sh#L12).
+Ensure that Python, Node, Docker, Docker Machine, Docker Compose, and imagemagick are installed. If you are using a mac with homebrew, you can run:
+
+`sh homebrew-deps.sh`
+
+Then:
 
 ```
 git clone git@github.com:cfpb/cfgov-refresh.git
```

Comment:
Can't wait to try this, but just to point out, this currently has users running `homebrew-deps.sh` before checking out cfgov-refresh ;)